primary/src/garbage_collector.rs

```diff
@@ -57,9 +61,19 @@ impl<PublicKey: VerifyingKey> GarbageCollector<PublicKey> {
     async fn run(&mut self) {
         let mut last_committed_round = 0;
         while let Some(certificate) = self.rx_consensus.recv().await {
-            // TODO [issue #9]: Re-include batch digests that have not been sequenced into our next block.
-
             let round = certificate.round();
+
+            // Re-include batch digests that have not been sequenced into our next block.
+            if certificate.author() == name {
```

Comment:
I thought that the certificates coming form the `rx_consensus` are the ones that have been sequenced, aren't they?